### ISSUE TITLE
[Bug]: r_w conflict error on ci test.

### ISSUE THREAD

**Issue by Ariznawlll** [matrixorigin/matrixone#12660]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93):  5c29cee0cfb9e6cbe57d90a021d93e58a3437fc3
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior
secnario 1：
<img width="1373" alt="image" 
src="https://github.com/matrixorigin/matrixone/assets/108530700/02702750-6108-4158-b706-fb1c9adfefdc">

log: [e2e-linux-x86-reports.zip](https://github.com/matrixorigin/matrixone/files/13316076/e2e-linux-x86-reports.zip)

secnario 2：
<img width="1381" alt="image" src="https://github.com/matrixorigin/matrixone/assets/108530700/bb37b2a3-f550-437a-a5b3-8ca75ebbc133">

log：log is large，if you need it，please contact me.

**Comment by XuPeng-SH** [comment]
fixed by #12668

**Comment by Ariznawlll** [comment]
Verified！
